File: app/src/main/java/com/lis/player_java/data/room/model/AlbumDB.java
package com.lis.player_java.data.room.model;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class AlbumDB {
    @PrimaryKey
    public Long id;
    public Long ownerId;
    public String accessKey;
    public String title;
    public int year;
    public int musicCount;
    public Boolean isFollowing;
    public Boolean isExplicit;
    public String photo600;
    public String photo1200;

    public AlbumDB(Long id, Long ownerId, String accessKey, String title, int year, int musicCount,
                   boolean isFollowing, boolean isExplicit, String photo600, String photo1200) {
        this.id = id;
        this.ownerId = ownerId;
        this.accessKey = accessKey;
        this.title = title;
        this.year = year;
        this.musicCount = musicCount;
        this.isFollowing = isFollowing;
        this.isExplicit = isExplicit;
        this.photo600 = photo600;
        this.photo1200 = photo1200;
    }
}

File: app/src/main/java/com/lis/player_java/data/room/model/AlbumDB.kt
package com.lis.player_java.data.room.model

import androidx.room.PrimaryKey
import com.lis.player_java.data.room.model.AlbumForMusic
import com.lis.player_java.data.room.model.GenreType
import androidx.room.Embedded
import androidx.room.Entity
import com.lis.player_java.data.room.model.MusicDB
import com.lis.player_java.data.room.model.ArtistDB
import com.lis.player_java.data.room.model.ArtistType

@Entity
data class AlbumDB(
    @PrimaryKey val id: Long,
    val ownerId: Long,
    val accessKey: String,
    val title: String,
    val year: Int,
    val musicCount: Int,
    val isFollowing: Boolean,
    val isExplicit: Boolean,
    val photo300: String,
    val photo600: String
)